app/models/cohort.rb

```diff
@@ -82,6 +82,10 @@ def display_name
     start_year.to_s
   end
 
+  def payments_frozen?
```

Comment:
I've been referencing candidates with `eligible_to_change_cohort_and_continue_training?` - we should probably get on the same page here; I'm happy to adopt something like `eligible_to_change_cohort_due_to_payment_freeze?` if that makes sense?

Reply:
@ethax-ross I think method names and fields should be reduced only to the domain of the class where they sit. 
I mean, a cohort gets their payments frozen or not.

Then what we do with participants or any other cohort related entity due to a cohort having frozen payments is to me something outside the scope of the Cohort and therefore out of it.

Let's see what other people think about it

Reply:
I think it makes sense to be consistent with the cohort naming; its a different model but it directly relates to this method (I'll be updating my method once this goes in; at the moment I just assume cohorts > 3 years older than the current are closed/payments frozen).

Reply:
For me the `eligible_to_change_cohort_` method prefix doesn't really make sense for an instance of a `Cohort` record as it's attempting to explain the state of a relationship between two cohort records. 
`Cohort#payments_frozen?` is intuitive and feels like the sort of naming convention we employ in other services.